internal/gitlab/gitlab.go

```diff
@@ -1725,3 +1725,16 @@ func TodoMarkAllDone() error {
 	}
 	return nil
 }
+
+func TodoMRCreate(project string, mrNum int) (int, error) {
+	p, err := FindProject(project)
+	if err != nil {
+		return 0, err
+	}
+
+	todo, _, err := lab.MergeRequests.CreateTodo(p.ID, mrNum)
+	if err != nil {
+		return 0, err
+	}
```

Comment:
Based on my last comment, we should not ignore `resp` return code here: GitLab API states it returns `HTTP 304 Not Modified` for already existent to-do items, in this case we should check `resp.StatusCode == http.StatusNotModified` and return a new error.

Reply:
Fixed as well.